Makefile

```diff
@@ -46,6 +46,9 @@ SUFFIX       :=
 INSTALL      := install
 prefix       := /usr/local
 bindir       := $(prefix)/bin
+NCPUS        = $(shell $(LOCAL_BIN)/ncpus)
+
+MAKEFLAGS += --jobs=$(NCPUS)
```

Comment:
What about:
```make
PARALLEL = -j$(NCPUS)
MAKEFLAGS += $(PARALLEL)
```

So that the user can override on the command line with `make PARALLEL=`